### PR TITLE
#3933 - Add comments in order

### PR DESCRIPTION
**Pull request by IrinaZhadzinets**
**Related issue(s):**
* https://github.com/scandipwa/scandipwa/issues/3933
* https://github.com/scandipwa/sales-graphql/pull/5

**Problem:**
* Lack of comments on the order on the frontend

**In this PR:**
* Add comment in order in my account if user left comment in Admin panel

### CODE REVIEW

**Review by AleksandrsKondratjevs — APPROVED** [review]
@aleksandrakorolova,

Won't in this PR also fit comments for shipment and invoices (if they can be rendered on FE)?

**Comment by aleksandrakorolova** [comment]
@AleksandrsKondratjevs 
You are right! We can set visible comments for shipment, invoice, and refund

@IrinaZhadzinets 
Please, add comments  for shipment, invoice, and refund

File: packages/scandipwa/src/component/MyAccountOrderItemsTable/MyAccountOrderItemsTable.style.scss [hunk 1]
```diff
@@ -98,4 +98,29 @@
...
         width: 15%;
     }
+
+    &-Comments {
+        &Title {
+            font-size: 18px;
+            font-weight: 600;
+            line-height: 1.1;
+            margin-bottom: 10px;
+            margin-top: 20px;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/MyAccountOrderItemsTable/MyAccountOrderItemsTable.style.scss:108]: Use logic properties

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
Nice! Just some minor changes required

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**